dmoj/cptbox/helper.cpp

```diff
@@ -306,7 +306,7 @@ int memory_fd_create(void) {
 #ifdef __FreeBSD__
```

Comment:
Actually, is this function called on FreeBSD anymore? Are you creating the tempfile in Python instead?

Reply:
Let's keep this around for now, since I'd rather this function work on all platforms, as the detection logic for the FreeBSD case is now different. If FreeBSD implements `/proc/[pid]/fd` some day, this will magically work.